File: app/src/main/java/com/android/timlin/ivedioplayer/file/data/FileEntry.java
package com.android.timlin.ivedioplayer.file.data;

import java.io.Serializable;

/**
 * SD卡上的视频信息存放类
 * Created by linjintian on 2019/2/17.
 */
public class FileEntry implements Serializable {
    /**
     * 文件夹路径
     **/
    public String path;
    /**
     * 文件夹里有多少个视频文件
     **/
    public int count;
    private static final long serialVersionUID = 1L;
    public String name;

    public FileEntry() {
    }

    public FileEntry(String path, int count) {
        this.path = path;
        this.count = count;
    }

    public FileEntry(String path, String name, int count) {
        this.path = path;
        this.name = name;
        this.count = count;

    }
}

File: app/src/main/java/com/android/timlin/ivedioplayer/list/file/FileEntry.kt
package com.android.timlin.ivedioplayer.list.file

import java.io.Serializable

/**
 * SD卡上的视频信息存放类
 * Created by linjintian on 2019/2/17.
 */
class FileEntry : Serializable {
    /**
     * 文件夹路径
     */
    var path: String
    /**
     * 文件夹里有多少个视频文件
     */
    var count: Int = 0
    var name: String

    constructor(path: String, name: String, count: Int) {
        this.path = path
        this.name = name
        this.count = count
    }

    companion object {
        private const val serialVersionUID = 1L
    }
}